src/main/resources/static/assets/js/sdh.js

```diff
@@ -47,9 +24,44 @@ class ModalEditRow {
        	  <template slot="flagged" slot-scope="row">
        	  	<span class="flag" :class="{'flag-enabled':(row.item.flagged)}" @click="toggleFlag(row)"><i class="fas fa-exclamation-circle"></i></span>
       	  </template>
-     	  <template slot="edit" slot-scope="row">
-      	  	<i class="far fa-edit" @click="edit(row.item, row.index, $event.target)"></i>
+       	  <template slot="clinician_priority" slot-scope="row">
+      	  	<div v-if="row.index === rowToEdit">
+  				<b-form-group>
+  					<b-form-select v-model="row.item.clinician_priority" :options="priorityOptions" />
+  				</b-form-group>
+			</div>
+			<div v-else>{{row.item.clinician_priority}}</div>
+      	  </template>
+      	  <template slot="patient_readiness" slot-scope="row">
+      	  	<div v-if="row.index === rowToEdit">
+  				<b-form-group>
+  					<b-form-select v-model="row.item.patient_readiness" :options="readinessOptions" />
+  				</b-form-group>
+			</div>
+			<div v-else>{{row.item.patient_readiness}}</div>
       	  </template>
+      	  <template slot="referred" slot-scope="row">
+      	  	<div v-if="row.index === rowToEdit">
+  				<b-form-group>
+  					<b-form-select v-model="row.item.referred" :options="yesNoOptions" />
+  				</b-form-group>
+			</div>
+			<div v-else>{{_selectText(yesNoOptions, row.item.referred)}}</div>
+      	  </template>
+      	  <template slot="referral_complete" slot-scope="row">
+      	  	<div v-if="row.index === rowToEdit">
+  				<b-form-group>
+  					<b-form-select v-model="row.item.referral_complete" :options="yesNoOptions" />
+  				</b-form-group>
+			</div>
+			<div v-else>{{_selectText(yesNoOptions, row.item.referral_complete)}}</div>
+      	  </template>
+    	  <template slot="edit" slot-scope="row">
+      	  	<div v-if="row.index === rowToEdit">
+				<i class="fas fa-save fa-lg" @click="submitClinicianReview(row.item)"></i>
```

Comment:
Building off of Heath's comment about matching the color, you could probably add the `text-primary` class here too.

Reply:
Ah, I see. That is a better way to do it than specifying the color in css.